tests/integration/traces/test_traces_command.py

```diff
@@ -0,0 +1,235 @@
+import itertools
+import time
+from pathlib import Path
+from typing import List
+from unittest import skipIf
+
+import boto3
+
+from samcli.lib.observability.util import OutputOption
+from tests.integration.deploy.deploy_integ_base import DeployIntegBase
+from tests.integration.traces.traces_integ_base import TracesIntegBase, RETRY_COUNT, RETRY_SLEEP
+from tests.testing_utils import run_command, RUNNING_ON_CI, RUNNING_TEST_FOR_MASTER_ON_CI, RUN_BY_CANARY, \
+    method_to_stack_name, kill_process, start_persistent_process, read_until
+from datetime import datetime
+import logging
+from parameterized import parameterized
+
+LOG = logging.getLogger(__name__)
+
+SKIP_TRACES_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
+
+
+@skipIf(SKIP_TRACES_TESTS, "Skip traces tests in CI/CD only")
+class TestTracesCommand(TracesIntegBase):
+    @classmethod
+    def setUpClass(cls):
+        cls.lambda_client = boto3.client("lambda")
+        cls.cfn_client = boto3.client("cloudformation")
+        cls.xray_client = boto3.client("xray")
+
+        cls.test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "traces")
+
+        cls.stack_name = method_to_stack_name("test.integration.test_traces_command")
```

Comment:
What would be the `stack_name` here? It should start with `test` so that it can be cleaned up afterwards. Can you confirm if the name would be correct. We can also think about changing `method_to_stack_name` to add `test` prefix if it doesn't exist.

Reply:
Stack name start with test-traces-command-xxxxxx, it should be ok. I also check the log integration test and found it is not start with `test`. We may need to update the log one.